cernopendata/templates/cernopendata_records_ui/docs/detail.html

```diff
@@ -1,64 +1,62 @@
 {%- extends "invenio_records_ui/detail.html" %}
 
 {%- block page_body %}
-<div class="container-fluid background">
-    <div class="container detail-page" ng-app="recordApp">
-        <div class="row">
-            <div class="{% if record.related %} col-md-10 {% else %} col-md {% endif %}">
-                <div class="card card-style">
-                    <div class="card-body">
-                        <h1 class="d-inline"> {{ record.title }} </h1>
-                        <p>
-                        {% if record.get('created') %} <small>{{ record.get('created') }} by </small> {% endif %}
-                        <small>{{ record.author}} </small>
-                        </p>
-                        <p class="badges-box">
-                        {% if record.type %} <a class="badge badge-type" href="/search?type={{ record.type.primary }}">{{ record.type.primary }}</a>
-                        {% if record.type.secondary %}
-                        {% for type in record.type.secondary %}
-                        <a class="badge badge-subtype" href="/search?type={{ record.type.primary }}&subtype={{ type }}">{{ type }}</a>
-                        {% endfor %}
-                        {% endif %}
+<div class="ui {% if record.related %}two{% else %}one{% endif %} centered column stackable grid container detail-page">
+    <div class="row">
+        <div class="{% if record.related %}thirteen wide column{% else %}sixteen wide column{% endif %}">
+            <div class="ui segment">
+                <div class="card-body">
```

Comment:
@mvidalgarcia I left this `card-body` class as is because I saw usages in search, however, now that https://github.com/cernopendata/opendata.cern.ch/issues/3014 is out, I see we don't really need it (just asking for confirmation). If that is the case, then I can perhaps amend it.

Reply:
It's being used [here](https://github.com/cernopendata/opendata.cern.ch/pull/3020/files#diff-e57c82b46c275a58fdeb7b4dc6500177cc1afbe84ec909d3f861e38c58d24d4cR9) so we need it.